### ISSUE TITLE
Error wn-iconbitmap

### ISSUE THREAD

**Issue by fermani10** [FlubOtic/Spotify-Playlist-Downloader#1]
Hello, how can I solve this error?
![image](https://user-images.githubusercontent.com/31140793/117437114-29bf4e80-af06-11eb-8d3b-225390b77f00.png)

From already thank you very much

**Comment by FlubOtic** [comment]
There is one more line you will need to remove if you chose not to download
the icon:
icon = os.path.dirname(__file__) + "\\Images\\Spotify Playlist To MP3
Icon.ico"

Sorry I didn't see it.

Hopefully it works afterward. Please tell me if not. Thanks

**Comment by fermani10** [comment]
Bro when looking for the link it throws me, INVALID_CLIENT: Invalid redirect URI    , I use http://localhost:8888/callback/

**Comment by FlubOtic** [comment]
Make sure the variable in main.py, "SPOTIPY_REDIRECT_URI" is set to the
string value: "http://localhost:8888/callback/"

You also need to go to your Spotify App (
https://developer.spotify.com/dashboard/applications) and click "Edit
Settings", you will need to set "Redirect URIs" to
http://localhost:8888/callback/ (Don't forget to press save)
The link has to end with a slash "/"

Also make sure your variables SPOTIPY_CLIENT_ID and SPOTIPY_CLIENT_SECRET
are set to the correct values (Both are on the front page of your Spotify
App).

If you have any problems with it afterwards let me know.

**Comment by fermani10** [comment]
With this information that you fix, it is not necessary on the new computer to log into spotify. There the application would work from any computer?

**Comment by FlubOtic** [comment]
It should work on any computer, but make sure no one sees your
SPOTIPY_CLIENT_SECRET value. It needs to stay secret.

**Comment by fermani10** [comment]
Hello, can you implement a button to make one again after any download?

a button that returns to the beginning.

**Comment by FlubOtic** [comment]
Yes I'll implement it right now. I should be done within 3 hours (Hopefully
an hour). I'll notify you when it's done.

**Comment by fermani10** [comment]
Thank you very much, you are a teacher to me. I learned a lot with all this

**Comment by FlubOtic** [comment]
I'm glad I could help, I didn't expect anyone to even use my code :)

I should have the button uploaded in about an hour.

**Comment by FlubOtic** [comment]
I finished early. I added a button that says "Restart" after you finish
downloading. It should work, if you run into any issues just let me know.

**Comment by fermani10** [comment]
Thanks ,  When downloading a playlist of 86 songs, it might stop in the process, right?

**Comment by FlubOtic** [comment]
What do you mean by stop? Does it stop temporarily?

**Comment by fermani10** [comment]
Yes if it stops temporarily , 21/86 tracks

**Comment by FlubOtic** [comment]
Do you get an error message in the terminal? If not, it is probably
downloading a bigger music file than usual. How long does it stop for?

**Comment by fermani10** [comment]
![image](https://user-images.githubusercontent.com/31140793/117506939-c9a7c700-af5c-11eb-9b8e-08e0a28ae998.png)
![image](https://user-images.githubusercontent.com/31140793/117506986-d9271000-af5c-11eb-8c3e-140d2f58d553.png)
an existing connection was forced to break by the remote host. 

that's the error that appears in Spanish.

**Comment by FlubOtic** [comment]
Do you only get this error when it stops? How long does it take for it to
start again?

I can't find much information on your error. I think it has to do with your
personal internet connection. I will update the code to use less internet.

**Comment by fermani10** [comment]
Yes if this error arises when it stops. I leave the playlist for you to try from the console and throw the error in English.  
It runs wonderfully, the only problem is that it stops at a certain moment. I tried with 2 different internet and one with 600 megabytes. 

https://open.spotify.com/playlist/2Ue1mG81VZE0NNea9yrdbE

**Comment by FlubOtic** [comment]
I updated the code. I'm not sure if it will fix your error because I can't
test it. Please let me know if it fixes the issue. I have one more idea if
it still doesn't work.

**Comment by FlubOtic** [comment]
Sorry I couldn't help, good luck finding a solution.
